autopilot/contractor.go

```diff
@@ -221,10 +226,16 @@ func (c *contractor) performContractMaintenance(ctx context.Context, w Worker) (
 		return false, err
 	}
 
-	// min score to pass checks.
+	// score all hosts
+	scoredHosts, unusableHosts, err := c.scoreHosts(ctx, hosts, usedHosts, hostData, math.SmallestNonzeroFloat64) // avoid 0 score hosts
```

Comment:
I feel like `candidateHosts` is the more fitting name here for something that doesn't just score but also filter hosts good enough for formation.